desktop/cmp/contextmenu/StoreContextMenuItem.js

```diff
@@ -56,4 +52,19 @@ export class StoreContextMenuItem {
         this.prepareFn = prepareFn;
         this.recordsRequired = recordsRequired;
     }
-}
\ No newline at end of file
+}
+
+/**
+ * @callback storeCtxMenuItemActionFn - called on store context menu item click.
```

Comment:
interesting.

Reply:
Cool -- Wonder if we could discuss the convention for this callback type  name?  Wondering if capital casing would be more appropriate.  Also, is this complete unscoped, name wise?

Reply:
How about `Cb` vs. `Fn` at the end? 

Scoping of the names follows the general rules for JsDoc, of course, which is to say your guess is as good as mine as to how these things get resolved. They should come up with some kind of scheme for package names - could use something super-clever like your company's internet domain name, but maybe switch the order around so the last part comes first and then...oh wait...

You will see I am experimenting in one or two places with the `@alias` tag to see how we could attach more specific / reliable names to some classes with very general name - `@alias HoistColumn`.  What matters to me in the end is what the tooling we care about does with such things. My next step as soon as I can breathe again is to work on jsDoc > md conversion to generate docs for publishing, and in my mind the format and readability/linkability of those docs will be a big input into what tags/names we use and if they are effective or not.

Reply:
Swapping to `Cb` at end now, but realizing we might want to think about the constructor args themselves a bit more - we have `action` and `prepareFn` - not exactly consistent there.....